src/CMakeLists.txt

```diff
@@ -460,10 +465,10 @@ add_dependencies(${EXECUTABLE_HELLOREGION} ${COMMON_LIBS})
 set(EXECUTABLE_PROTOTEST prototest)
 add_executable(${EXECUTABLE_PROTOTEST} examples/prototest.cpp)
 target_link_libraries(${EXECUTABLE_PROTOTEST}
-                        ${COMMON_LIBS})
+                    ${COMMON_LIBS})
```

Comment:
Probably your Visual Studio is set to use tabs instead of spaces. This makes this weird indentation.